libpod/container_config.go

```diff
@@ -163,6 +163,8 @@ type ContainerRootFSConfig struct {
 	// Volatile specifies whether the container storage can be optimized
 	// at the cost of not syncing all the dirty files in memory.
 	Volatile bool `json:"volatile,omitempty"`
+	// ManagePasswd allows to user to override podman's passwd/group file setup
+	ManagePasswd bool `json:"manage_password,omitempty"`
```

Comment:
Why ManagePasswd as opposed to passwd?

Reply:
I thought it was more descriptive than just passwrd especially in the config, I can change it back if you disagree.

Reply:
You mean for podman inspect?

Reply:
yep, just realized I did not add a way to see this in inspect, I will add that now